client/src/components/Selection.jsx

```diff
@@ -0,0 +1,20 @@
+import { useState } from 'react'
+import Search from './Search.jsx'
+import SongList from './SongList.jsx'
+export function Selection() {
+  const [selected, setSelected] = useState([])
+  const addToSelected = (song) => {
+    setSelected([...selected, {...song, songID: crypto.randomUUID() }])
+  }
+  const handleRemove = (songID) => {
+    let newList = selected.filter((song) => songID != song['songID'])
+    setSelected(newList)
+  }
+
+  return (
+    <>
+      <Search addToSelected={addToSelected} />
```

Comment:
It's pretty minor, but the names here could be tightened up. You can have your own scheme... but I wasn't seeing the scheme here.

I try to use `add`, `handle`, `on`, in precise ways. Here's my rules:

For functions that do something, like remove or add a song, I'd keep the active verb... therefore `addSong` and `removeSong`. It's not named after the event handled.

For properties to hold event handlers, I stick with `on<Event>`. And for event handlers themselves, `handle<Event>`. This is to because the component doesn't need to, nor should it, control what is done in a response for an event. It just notifies its container what happened... therefore `onSongSelected`...